### ISSUE TITLE
Skip link is shown on backwards tab

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#138]
Screengrab/videos (insert file names here)
2020 08 18, Skip link is shown on backwards tab.mp4

Link
https://d8-stg.boston.gov/

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > macOS 10.15.6 > Chrome 84

Trusted Tester Test
Keyboard Focus

Brief Description
The skip link is shown when tabbing backwards

Steps to reproduce
1)	00:00, Load the page
2)	00:09, Navigate with the keyboard Tab key until the side menu is expanded
3)	00:12, Start clicking shift-Tab in order to navigate backwards
4)	00:15, The #Content link is shown when tabbing backwards, and not when tabbing forwards.
5)	00:17, Press the Enter key when the navigation is on the #Content link
a)	BUG:  The display is incorrect with the side menu partially shown and the menu close box is mis-aligned.

Desired Result
•	Skip link is not shown when tabbing backwards

Actual Result
•	The skip link is shown when tabbing backwards

**Comment by subaha-cob** [comment]
@andrebtlima and @jill-iterators We want the skip link to show just not with the menu covering it. 

Fixed the bug with menu covering the Skiplink. 

Test at: https://d8-uat.boston.gov/

**Comment by jill-iterators** [comment]
<img width="1534" alt="2020-08-27_09-30-07" src="https://user-images.githubusercontent.com/53226162/91448569-08bb3f80-e848-11ea-8b40-a8cba04aa3ab.png">

Please see image attached, highlighted both the content link and the black area at the top of the page. This corresponds to the video at 17 seconds.

**Comment by jill-iterators** [comment]
Retested on Incognito and this is fixed.

**Comment by david-iterators** [comment]
Here is the video for the original bug report.  Please note the bug at 00:17
[2020 08 18, Content link interferes with menu.zip](https://github.com/CityOfBoston/Iterators/files/5138530/2020.08.18.Content.link.interferes.with.menu.zip)

**Comment by jill-iterators** [comment]
We retested this on UAT with Chrome Incognito and this is fixed and verified.

**Comment by andrebtlima** [comment]
@subaha-cob this isn't working for me on prod

**Comment by subaha-cob** [comment]
@andrebtlima wait until Drupal deploy and test again. Some fixes where for boston.gov only.
